XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -344,7 +344,7 @@ Specifies the flags for this transaction. In addition to the universal transacti
 
 The value specifies the fee to charged by the issuer for secondary sales of the Token, if such sales are allowed. Valid values for this field are between 0 and 50,000 inclusive, allowing transfer rates of between 0.000% and 50.000% in increments of 0.001.
 
-The field MUST NOT be present if the `tfMPTCanTransfer` flag is not set. If it is, the transaction should fail and a fee should be claimed.
+The field MUST NOT be present if the `tfMPTCanTransfer` flag is not set. If it is, the transaction should fail with `temMALFORMED`.
```

Comment:
The proposed change makes sense to me. However, it has me wondering -- what should happen if `tfMPTCanTransfer` is set, but the fee is missing? Do we assume 0 fee?

Reply:
Yes, the fee would be zero in this case.